File: wallet-service/src/main/java/org/example/walletservice/controller/WalletController.java
package org.example.walletservice.controller;

import org.example.walletservice.dto.WalletResponse;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class WalletController {

    @GetMapping("/wallet")
    public WalletResponse getWallet() {
        return new WalletResponse("Sarthak", "10 Rs");
    }

}

File: wallet-service/src/main/kotlin/org/example/walletservice/controller/WalletController.kt
package org.example.walletservice.controller

import org.example.walletservice.dto.WalletResponse
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class WalletController {

    @GetMapping("/wallet")
    fun getWallet(): WalletResponse {
        return WalletResponse("Sarthak", "10 Rs")
    }

}

